automata/symbol_embedding/vector_databases.py

```diff
@@ -52,6 +58,31 @@ def add(self, entry: V) -> None:
             embeddings=[[int(ele) for ele in entry.vector]],
         )
 
+    def batch_add(self, entries):
+        """
+        Batch add entries to the database.
+
+        Arguments:
+        entries -- list of entries to add
+        """
+        documents = []
+        metadatas = []
+        ids = []
+        embeddings = []
+
+        for entry in entries:
+            documents.append(entry.document)
+            metadatas.append(entry.metadata)
+            ids.append(self.entry_to_key(entry))
+            embeddings.append([int(ele) for ele in entry.vector])
```

Comment:
this is good that we're passing the embeddings to chroma. 
We need to keep in mind not to exceed Openai's rate limits while creating embeddings. (not related to this PR since you're passing embeddings to chroma directly)